flytekit/image_spec/image_spec.py

```diff
@@ -67,6 +67,12 @@ class ImageSpec:
             If the option is set by the user, then that option is of course used.
         copy: List of files/directories to copy to /root. e.g. ["src/file1.txt", "src/file2.txt"]
         python_exec: Python executable to use for install packages
+        runtime_packages: List of packages to be installed during runtime. `runtime_packages` requires `pip` to be installed
```

Comment:
Can we add a comment telling people to not use this?  Or use it only as a last resort?